cmd/conduit/internal/print_utils.go

```diff
@@ -70,3 +72,121 @@ func DisplayProcessors(processors []*apiv1.Processor, indent int) {
 		fmt.Printf("%sUpdated At: %s\n", Indentation(indent+2), PrintTime(p.UpdatedAt))
 	}
 }
+
+// FormatLongString splits a string into multiple lines depending on the maxLineLength.
+func FormatLongString(paragraph string, maxLineLength int) string {
+	if len(paragraph) <= maxLineLength {
+		return paragraph
+	}
+
+	var result strings.Builder
+	var currentLine strings.Builder
+	words := strings.Fields(paragraph)
+	for _, word := range words {
+		// check if adding the next word would exceed the line length
+		if currentLine.Len()+len(word)+1 > maxLineLength {
+			result.WriteString(currentLine.String() + "\n")
+			currentLine.Reset()
+		}
+		if currentLine.Len() > 0 {
+			currentLine.WriteString(" ")
+		}
+		currentLine.WriteString(word)
+	}
+
+	// add the last line if it's not empty
+	if currentLine.Len() > 0 {
+		result.WriteString(currentLine.String())
+	}
+
+	return result.String()
+}
+
+func DisplayConfigParams(cfg map[string]*configv1.Parameter) {
+	table := simpletable.New()
+
+	table.Header = &simpletable.Header{
+		Cells: []*simpletable.Cell{
+			{Align: simpletable.AlignCenter, Text: "NAME"},
+			{Align: simpletable.AlignCenter, Text: "TYPE"},
+			{Align: simpletable.AlignCenter, Text: "DESCRIPTION"},
+			{Align: simpletable.AlignCenter, Text: "DEFAULT"},
+			{Align: simpletable.AlignCenter, Text: "VALIDATIONS"},
+		},
+	}
+
+	// create slices for ordered parameters, needed to keep the name
+	var requiredParams, otherParams, sdkParams []struct {
+		name  string
+		param *configv1.Parameter
+	}
+
+	// separate parameters into three groups for ordering purposes
+	for name, param := range cfg {
+		switch {
+		case strings.HasPrefix(name, "sdk"):
+			sdkParams = append(sdkParams, struct {
+				name  string
+				param *configv1.Parameter
+			}{name: name, param: param})
+		case isRequired(param.Validations):
+			requiredParams = append(requiredParams, struct {
+				name  string
+				param *configv1.Parameter
+			}{name: name, param: param})
+		default:
+			otherParams = append(otherParams, struct {
+				name  string
+				param *configv1.Parameter
+			}{name: name, param: param})
+		}
+	}
+
+	// combine ordered parameters
+	orderedParams := append(requiredParams, otherParams...) //nolint:gocritic // intentional
+	orderedParams = append(orderedParams, sdkParams...)
+
+	for _, item := range orderedParams {
+		r := []*simpletable.Cell{
+			{Align: simpletable.AlignLeft, Text: item.name},
+			{Align: simpletable.AlignLeft, Text: formatType(item.param.GetType().String())},
+			{Align: simpletable.AlignLeft, Text: FormatLongString(item.param.Description, 100)},
+			{Align: simpletable.AlignLeft, Text: item.param.Default},
+			{Align: simpletable.AlignLeft, Text: formatValidations(item.param.Validations)},
+		}
+		table.Body.Cells = append(table.Body.Cells, r)
+	}
+
+	table.SetStyle(simpletable.StyleDefault)
```

Comment:
I noticed this one sets a different style of a table. I don't dislike this one, but we should be consistent. Suggesting the ones we've used for now until we agreed on changing the rest.

```suggestion
	table.SetStyle(simpletable.StyleCompact)
```

Reply:
change the rest to `StyleDefault `